app/routes/missions.fermi/route.mdx

```diff
@@ -36,6 +38,25 @@ import logo from './logo.svg'
 
 </div>
 
+<WithFeature GBM_JSON>
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+Fermi/GBM distributes JSON-serialized notices for the detection of gamma-ray transients on the GCN Kafka topic `gcn.notices.fermi.gbm`.
+
+For schemas and examples see [/schema/stable/gcn/notices/fermi/gbm](/schema/stable/gcn/notices/fermi/gbm) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/gbm/).
+
+Detailed description and examples of Fermi/GBM Notices are available in the [GCN Schema GitHub project](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/fermi/gbm).
+
+<div className="overflow-table">
+
+| Type                          | Contents                               | Latency    |
+| ----------------------------- | -------------------------------------- | ---------- |
+| `gcn.notices.fermi.gbm.alert` | Localization, Count Rate, Significance | ~5 seconds |
```

Comment:
What about the strict topic?

Reply:
This is the only GBM Alert topic. We don't want "strct" topic(s). Please see discussion in the schema pull request: https://github.com/nasa-gcn/gcn-schema/pull/199#discussion_r1845270209

Reply:
That's for team use only, it wouldn't be public or streamed, so no point of mentioning to the community.